capa/features/extractors/cape/thread.py

```diff
@@ -40,14 +40,14 @@ def extract_call_features(behavior: Dict, ph: ProcessHandle, th: ThreadHandle) -
         if call["thread_id"] != tid:
             continue
 
-        caller = int(call["caller"], 16)
-        caller = AbsoluteVirtualAddress(caller)
+        # TODO this address may vary from the PE header, may read actual base from procdump.pe.imagebase or similar
+        caller = DynamicAddress(call["id"], int(call["caller"], 16))
         yield API(call["api"]), caller
-        for arg in call["arguments"]:
+        for n, arg in enumerate(call["arguments"]):
             try:
-                yield Number(int(arg["value"], 16)), caller
+                yield Number(int(arg["value"], 16), description=f"arg{n}"), caller
             except ValueError:
-                yield String(arg["value"]), caller
+                yield String(arg["value"], description=f"arg{n}"), caller
```

Comment:
return_address doesn't really make sense here, but it's the best we can do right now to tie this back to a VA